packages/gatsby/src/utils/node-manifest.ts

```diff
@@ -363,7 +387,16 @@ export async function processNodeManifests(): Promise<Map<
 
   const processNodeManifestQueue = fastq(processNodeManifestTask, 25)
 
-  for (const manifest of nodeManifests) {
+  if (totalManifests > NODE_MANIFEST_FILE_LIMIT) {
+    nodeManifests = [...nodeManifests]
+    nodeManifests.sort(nodeManifestSortComparerAscendingCreatedAt)
+  }
+
+  for (const [i, manifest] of nodeManifests.entries()) {
+    if (i >= NODE_MANIFEST_FILE_LIMIT) {
+      break
+    }
+
     processNodeManifestQueue.push(manifest, () => {})
```

Comment:
So we just stop creating manifest files? Wouldn't that be an issue?

Reply:
Once we're creating more than 100 manifests it's very unlikely those manifests were intentionally created. It should be 1 manifest per user edit in the CMS but in cases where every piece of content was updated recently (like when a site is migrated or all pieces of content are programatically updated) you end up with potentially 1 manifest per piece of content in the CMS which is the issue we're running into here.
So the solution is to cap the total

Reply:
👍 should we lower the limit to like 500 or something?